boards/arm/numaker_pfm_m467/numaker_pfm_m467_defconfig

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_M46X=y
+CONFIG_SOC_M467=y
+CONFIG_BOARD_NUMAKER_PFM_M467=y
+CONFIG_PINCTRL=y
+CONFIG_GPIO=y
+CONFIG_GPIO_NUMAKER=y
+
+# Enable system clock controller driver
+CONFIG_CLOCK_CONTROL=y
+CONFIG_CLOCK_CONTROL_NUMAKER_SCC=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=200000000
+
+# enable uart driver
+CONFIG_SERIAL=y
+CONFIG_UART_NUMAKER=y
```

Comment:
redundant

Reply:
It's OK to remove CONFIG_UART_NUMAKER=y.
But need keep CONFIG_SERIAL=y, else hello_world sample can't link uart_numaker.o .
Is it correct ?

Reply:
Yes, only driver class is needed, other is automatically selected by DT status.